.gitignore

```diff
@@ -19,3 +19,4 @@ target
 *.pyc
 flink-ml-python/dist/
 flink-ml-python/apache_flink_ml.egg-info/
+flink-ml-python/venv/
```

Comment:
IMO, we don't need to add this.  Putting venv in the source directory is not a standard way